extensions/src/data_bridge/index.ts

```diff
@@ -39,15 +58,14 @@ const plugin: JupyterFrontEndPlugin<void> = {
             );
         };
 
-        // TODO: set type for data
         window.addEventListener("message", async (event) => {
             if (event.data.type === "from-host-to-iframe") {
                 let data = event.data.data;
                 const dataJson = JSON.stringify(data);
                 const code = `
-  import json
-  data = json.loads('${dataJson}')
-  `;
+    import json
```

Comment:
Prob no need for the extra indent

Reply:
It was autoformatting, shouldn't affect behavior